shells/dev/target/NativeTypes.vue

```diff
@@ -11,43 +11,116 @@
     <TestComponent ref="component" />
 
     <p>
-      <button @click="sendComponent">Vuex mutation</button>
+      <button @click="sendComponent()">Vuex mutation</button>
+      <button @click="createLargeArray()">Create large array</button>
+    </p>
+
+    <h3>Set</h3>
+    <pre>{{ setDisplay() }}</pre>
+
+    <h3>Map</h3>
+    <pre>{{ mapDisplay() }}</pre>
+
+    <p>
+      <button @click="testVuexSet()">Vuex Set</button>
+      <button @click="testVuexMap()">Vuex Map</button>
+      <button @click="forceRefresh()">Refresh</button>
     </p>
   </div>
 </template>
 
 <script>
 import { mapState, mapGetters, mapMutations } from 'vuex'
+import CompDef from './Other.vue'
 
 export default {
   components: {
     TestComponent: {
-      data: () => ({ foo: '42' }),
       props: { bar: { default: 'hey' }},
+      data: () => ({ foo: '42' }),
+      computed: {
+        parentComp () { return this.$parent }
+      },
       render: h => h('div', '<TestComponent />')
     }
   },
+
   data () {
     return {
       localDate: new Date(),
-      testComponent: null
+      reg: /abc/gi,
+      testComponent: null,
+      hello: function foo (a, b, c) {},
+      hey: function empty () {},
```

Comment:
Can you also add anonymous function here and make sure it works good as well?
`hi: function (a, b) {}` should be displayed probably as  `f (a, b)`, also what about arrow functions? Or `hi() {}`?